### ISSUE TITLE
Setup Repository

### ISSUE THREAD

**Issue by joshterrell805** [joshterrell805/VisualTimerAndroid#1]
Tech Stack
- React Native & Redux
- TypeScript
- node & npm
- [React native elements](https://github.com/react-native-elements/react-native-elements)
- [Jest-TS](https://www.testim.io/blog/typescript-unit-testing-101/)? (testing)

Development environment
- Android Studio (for cli tools to build and install app) - https://reactnative.dev/docs/environment-setup
- VS Code?
- Ubuntu for Windows
- Git (force unix line endings)

What should be present when this issue is complete
- [x] gitignore
- [x] Working unit tests
- [x] Blank android app that displays hello world
- [x] Successfully able to install and run on my Android phone

**Comment by joshterrell805** [comment]
So far I've initialized a typescript app and I'm working at running it in an emulator before I try running it on my phone.

It's currently unable to locate the emulator I installed when I use the command `npx react-native run-android`

I've got AndroidStudio installed for windows, but I'm running `npx react-native` commands through ubuntu on windows. 

I've tried setting the ANDROID_HOME envvar to `/mnt/c/Users/josh/AppData/Local/Android/Sdk` but so far no luck.
